action/run_custom_test.go

```diff
@@ -39,6 +39,44 @@ func TestRunCustom(t *testing.T) {
 		assert.Equal(t, map[string]interface{}{"some-output": "SOME CONTENT"}, c.Outputs)
 	})
 
+	t.Run("configure operation", func(t *testing.T) {
+		c := newClaim()
+		d := &mockDriver{
+			shouldHandle: true,
+			Result: driver.OperationResult{
+				Outputs: map[string]string{
+					"/tmp/some/path": "SOME CONTENT",
+				},
+			},
+			Error: nil,
+		}
+		inst := &RunCustom{Driver: d, Action: "test"}
```

Comment:
I think we should factorize those tests, as they are exactly the same for all the actions (good hint is that this action is still called `inst` 😇 ).